parser/grammar.y

```diff
@@ -971,6 +973,14 @@ string_enumeration_item
     ;
 
 
+percent_expression
```

Comment:
`percent_expression` is not needed, right?

Reply:
I removed this clause in https://github.com/VirusTotal/gyp/commit/bd33cde90fa94656ddd6a9a7809a935a6609640a